docs/how-to/hip_porting_driver_api.rst

```diff
@@ -123,8 +120,8 @@ HIPIFY translation of CUDA driver API
 The HIPIFY tools convert CUDA driver APIs for streams, events, modules, devices, memory management, context, and the profiler to the equivalent HIP calls. For example, ``cuEventCreate`` is translated to ``hipEventCreate``.
```

Comment:
"The HIPIFY tools convert CUDA driver APIs for streams"

he HIPIFY tools convert CUDA driver APIs such as streams

Reply:
Fixed